emacs/radian.el

```diff
@@ -651,8 +681,8 @@ level of init.local.el."
              (boundp 'mac-command-modifier))
     (setq mac-option-modifier 'meta)
     (setq mac-command-modifier 'super))
-
-  (bind-key "s-z" #'undo-tree-undo)
+  (when (radian-enabled-p 'undo-tree)
+    (bind-key "s-z" #'undo-tree-undo))
```

Comment:
Should this maybe be moved to the config section of `(use-package undo-tree)`?

Reply:
I think we can actually bind `s-z` to `#'undo`, and rely on the fact that `undo-tree-map` remaps `undo` to `undo-tree-undo`. That would eliminate the problem.